.github/workflows/deploy.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         poetry run cdk deploy --all --require-approval never
       env:
         DEPLOY_ENV: nonprod
-        DATALAKE_SAML_IDENTITY_PROVIDER_ARN: ${{ secrets.DATALAKE_SAML_IDENTITY_PROVIDER_ARN }}
+        DATALAKE_SAML_IDENTITY_PROVIDER_ARN: ${{ secrets.DATALAKE_SAML_IDENTITY_PROVIDER_ARN_NON_PROD }}
```

Comment:
We probably should update the documentation around this at the same time.

Reply:
Documentation contains SAML configuration.

Reply:
But it refers to the original secret name, not these two.

Reply:
But it is the only one Data Lake configuration variable. Because we are deploying to different AWS where SAML ID provider ARN is different we have to use different values in our deployment process. But it doesn't have anything to do with Data Lake configuration as such.

Reply:
Gotcha, thanks!